chain/consensus/filcns/upgrades.go

```diff
@@ -296,6 +298,10 @@ func DefaultUpgradeSchedule() stmgr.UpgradeSchedule {
 			StopWithin:      10,
 		}},
 		Expensive: true,
+	}, {
+		Height:    build.UpgradeCalibrationDragonFixHeight,
+		Network:   network.Version22,
+		Migration: upgradeActorsV13VerifregFix(calibnetv13BuggyVerifregCID1, calibnetv13CorrectManifestCID1),
```

Comment:
Shouldn't this be from `calibnetv13BuggyManifestCID1 ` to ` calibnetv13CorrectManifestCID1` ?

Reply:
No, the params [here](https://github.com/filecoin-project/lotus/pull/11776/files#diff-31149461fe26df8474d8dd28e8228b2c53e91fb4afa9aecf77a5244fe6072e59R2315) aren't "buggy" and "correct" manifests, they're the buggy ACTOR cid (used for sanity-checking) and the correct manifest CID. See usage in function belowe.